"email_forwarding" addition

Nightsmore opened the pull request.

adityaarakeri reviewed (changes requested):
please do not include binary files in your PR.

Nightsmore commented:
> please do not include binary files in your PR.

I've removed the offending file!  Updated with the .psm1 to help create a config file for the script.
